tests/test_checkpoint.sh

```
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# STUB TEST for test_checkpoint. Replace with real tests.
+
+set -euo pipefail
+
+declare -A module_options
+
+if [[ "tools/stub_missing_src_tests.sh" == "tools/stub_missing_src_tests.sh" ]]; then
```

Shellcheck (0.10.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)